src/compiler/transformers/module/module.ts

```diff
@@ -545,14 +550,24 @@ namespace ts {
             //          : new Promise(function (_a, _b) { require([x], _a, _b); }); /*Amd Require*/
             // });
             needUMDDynamicImportHelper = true;
-            return createConditional(
-                /*condition*/ createIdentifier("__syncRequire"),
-                /*whenTrue*/ transformImportCallExpressionCommonJS(node),
-                /*whenFalse*/ transformImportCallExpressionAMD(node)
-            );
+            if (isSimpleCopiableExpression(arg)) {

```

Comment:
Its generally safer to _always_ clone (or cache in a temp at runtime) an expression that is used more than once, unless that expression is a `GeneratedIdentifier`.